Geometry/TrackerGeometryBuilder/test/BuildFile.xml

```diff
@@ -42,6 +42,7 @@
 
 <bin file="phase1PixelTopology_t.cu">
   <use name="HeterogeneousCore/CUDAUtilities"/>
+  <use name="Geometry/CommonTopologies"/>
```

Comment:
This could (should?) be
```suggestion
  <use name="Geometry/CommonTopologies" source_only="1"/>
```
because only a header (`SimplePixelTopology.h`) is used from that package.